src/pages/[platform]/build-a-backend/auth/concepts/multi-factor-authentication/index.mdx

```diff
@@ -44,31 +48,160 @@ export const auth = defineAuth({
 // highlight-start
   multifactor: {
     mode: 'OPTIONAL',
-    totp: true
+    totp: true,
```

Comment:
Do we need to callout that Email MFA cannot currently be configured with backend?

Reply:
yes, can we add a callout for it and point folks to this issue? https://github.com/aws-amplify/amplify-backend/issues/2159

```
**Note:** email-based MFA is currently not supported with `defineAuth`. We are working towards supporting this feature. For more information, visit the [feature request in GitHub](https://github.com/aws-amplify/amplify-backend/issues/2159)
```

Reply:
Added callout for backend support

Reply:
Added the CDK example to the[ Modify Amplify-generated Cognito resources with CDK](https://docs-email-mfa.dmhjrabi3pkql.amplifyapp.com/react/build-a-backend/auth/modify-resources-with-cdk/#override-cognito-userpool-multi-factor-authentication-options) page and added a link from the MFA page just below the callout.